app/assets/javascripts/lib/ranger_connection.js.coffee

```diff
@@ -1,15 +1,21 @@
 stream = ['global',"#{gon.market.id}", 'order', 'trade']
 query = {stream: stream}
 address = 'ws://' + "#{gon.ranger_host}:#{gon.ranger_port}?" + $.param(query, true)
-
-console.info('Connecting to Ranger at', address)
+barongPath = "#{gon.barong_domain}" + "/api/v1/sessions/jwt"
 window.ranger = new RangerWebSocket address
-
-ranger.bind 'open', ->
-  console.info 'Connection to Ranger at has been established'
-  return if not gon.user
-  console.info 'Solving authorization challenge by sending JWT token'
-  ranger.send JSON.stringify {jwt: "Bearer #{gon.user.token}"}
-
-ranger.bind 'error', ->
-	console.error(data)
+$.ajax({
+  type: "POST",
+  url: barongPath,
+  async: true,
+  xhrFields: {
+    withCredentials: true
+  },
+  success: (res) ->
+    ranger.bind 'open', ->
+      console.info 'Connection to Ranger at has been established'
+      return if not gon.user
```

Comment:
we don't need that check anymore

Reply:
why? if user not login we will not send call to the ranger

Reply:
Oh, I see the purpose of that check now.